homeassistant/components/sensor/fail2ban.py

```diff
@@ -43,7 +45,7 @@
     """Set up the fail2ban sensor."""
     name = config.get(CONF_NAME)
     jails = config.get(CONF_JAILS)
-    scan_interval = config.get(CONF_SCAN_INTERVAL)
+    scan_interval = config.get(CONF_SCAN_INTERVAL, DEFAULT_SCAN_INTERVAL)
```

Comment:
This is not the correct approach. Please see comment here:
https://github.com/home-assistant/home-assistant/pull/10500#discussion_r150186077

Reply:
I am not sure of how to do but I will try something.

Reply:
I will let you check the changes, but it works in my install.